packages/contrast-colors/index.js

```diff
@@ -207,32 +240,39 @@ function generateContrastColors({
   if (!colorKeys) {
     throw new Error(`Color Keys are undefined`);
   }
+  for (let i=0; i<colorKeys.length; i++) {
+    if (colorKeys[i].length < 6) {
+      throw new Error('Color Key has improper length');
+    }
+    else if (colorKeys[i].length == 6 && colorKeys[i].charAt(0) != 0) {
```

Comment:
Hmm, this is a little confusing. Are you trying to check if the first character is a `#` or a 0?

Reply:
Checking to see if color is 6 characters long and the first character is *not* a `#`